src/controllers/hid/hidenumerator.cpp

```diff
@@ -8,6 +8,16 @@
 namespace {
 
 bool recognizeDevice(const hid_device_info& device_info) {
+    // Skip mice and keyboards. Users can accidentally disable their mouse
+    // and/or keyboard by enabling them as HID controllers in Mixxx.
+    // https://bugs.launchpad.net/mixxx/+bug/1940599
+    if (device_info.usage_page == kGenericDesktopUsagePage &&
+            (device_info.usage == kGenericDesktopMouseUsage ||
+                    device_info.usage == kGenericDesktopKeyboardUsage)) {
+        return false;
+    }
```

Comment:
Can we move this to the default deny list instead? 
That would be the natural place.

Reply:
I tried that at first but the backwards logic made it really confusing. I think this implementation is more straightforward. First there is this check for all mice and keyboards, then the denlylist is used to filter specific device models.